src/Dapr.Client/DaprClient.cs

```diff
@@ -324,5 +324,17 @@ public async ValueTask<StateEntry<TValue>> GetStateEntryAsync<TValue>(string sto
             string key,
             Dictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Gets all secret values from the secret store.
+        /// </summary>
+        /// <param name="storeName">Secret store name.</param>
+        /// <param name="metadata">An key/value pair that may be consumed by the secret store.  This depends on the secret store used.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="ValueTask{T}" /> that will return the value when the operation has completed.</returns>
+        public abstract ValueTask<Dictionary<string, string>> GetBulkSecretAsync(
```

Comment:
Should this be `GetAllSecretsAsync()`? It's not doing any filtering, it's getting all of the ones you have access to.

Reply:
The `GetBulk` prefix was added to keep consistency with naming used by Dapr: https://github.com/dapr/components-contrib/pull/310#discussion_r504195884